keylime/tenant.py

```diff
@@ -543,7 +542,7 @@ def do_cvdelete(self):
             deleted = False
             for _ in range(12):
                 response = httpclient_requests.request("GET", "%s"%(self.cloudverifier_ip), self.cloudverifier_port, params=params, context=self.context)
-                if response.status == 404:
+                if response.status == 200 or 404:
```

Comment:
This doesn't look correct. You might want this instead:
`if response.status in (200, 404):`

Reply:
looks good, thanks @zedr